misc.go

```diff
@@ -0,0 +1,210 @@
+/*
+ * File: misc.go
+ *
+ * Description: Contains a number of helpful util functions.
+ *
+ */
+
+package main
+
+import (
+	"crypto/rand"
+	"encoding/base64"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/signal"
+	"strings"
+	"time"
+)
+
+//! Wrapper to make golang error funcs seem more C-like
+/*
+ * @param     string    ASCII to dump to stdout
+ *
+ * @return    error     error message, if any
+ */
+func errorf(ascii string) error {
```

Comment:
Originally I had planned on using them as aliases, and simply calling them "print()" or "error()" seemed too plain. That said, they are not actually needed, so I have since removed them.

https://github.com/rbisewski/lclusterd/commit/295c0c9c834f27bbac7a56b9b8a6cfcc8742c881